_data/resources.yml

```diff
@@ -331,3 +331,6 @@
     - url: http://tenon.io/
       title: "Accessibility Testing API"
       description: "By Tenon.io"
+    - url: https://packagecontrol.io/packages/WAI-ARIA
```

Comment:
I think Package Control link is more appropriate, since it is a Sublime Text plugin and Package Control is its dedicated package manager.